src/pages/auth/signIn/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { signInWithEmailAndPassword,getAuth } from 'firebase/auth';
+import "@/firebase"
+import styles from "@/styles/signIn.module.css"
+
+const SignIn = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSignIn =(e)=>{
```

Comment:
We need to save the token on the client so that when communicating with the backend we have the token in the requests.